core/src/main/java/org/axonframework/queryhandling/QueryBus.java

```diff
@@ -104,17 +89,68 @@
 
     /**
      * Dispatch the given {@code query} to a single QueryHandler subscribed to the given {@code query}'s
-     * queryName/initialResponseType/updateResponseType.
+     * queryName/initialResponseType/updateResponseType. The result is lazily created and until there is a subscription
+     * to initial result there will be no execution of query handler. In order not to miss updates, query bus will
```

Comment:
I think this sentence deserves additional `the`'s, at for example `...updates, the query bus...` and `...after the subscription query...`.